src/App.tsx

```diff
@@ -27,6 +33,7 @@ import {
   pathUsers,
 } from './constants/constants';
 import { useAppSelector } from './hooks/useAppSelector';
+import React from 'react';
 
 export const App: FC = () => {
   const isLoading = useAppSelector((state) => state.main.isLoading);
```

Comment:
create separate file for selectors, for example: 
`
const getIsLoading = (state: RootState): boolean => state.main.isLoading

usage: 
const isLoading = useAppSelector(getIsLoading)
`